<img src="https://coursesandconferences.wellcomeconnectingscience.org/wp-content/themes/wcc_courses_and_conferences/dist/assets/svg/logo.svg" width="300" height="50"> 

# Single Cell Genomics - Latin America & the Caribbean

## 9–15 August 2024, INCA, Rio de Janeiro, Brazil

[Wellcome Connecting Science Course Website Link](https://coursesandconferences.wellcomeconnectingscience.org/event/single-cell-genomics-latin-america-the-caribbean-20240809/) <br /> 
[Course Timetable 2024](https://github.com/WCSCourses/Single_Cell_Genomics_2024/blob/main/Single%20Cell%20LAM%202024%20Master%20Planner.xlsx%20-%20Final%20Timetable%2023%20Jul%202024-4.pdf)


## Summary

Single-cell sequencing technologies are powerful tools used to assess genomic, transcriptomic and proteomics information at the single-cell level. In recent years, the application of single-cell RNA sequencing (scRNA-seq) methodologies has become increasingly common across diverse fields of life sciences research. In medicine, single-cell approaches used to investigate organisms in both health and disease conditions have advanced unprecedented understanding of organisms’ biology as a whole. 

Successful scientific networks promoted by the [Human Cells Atlas](https://www.humancellatlas.org/) consortium highlighted the growing interest among scientists and an urgent need for developing capacity in single-cell technologies worldwide. However, there is still a lack of infrastructure, limited funding, and insufficient expertise to apply these cutting-edge technologies in Latin America. 

Aiming to expand the region’s capacity, our short course will offer hands-on laboratory and bioinformatics analysis training, tailored to empower researchers in Latin America with essential skills to employ single-cell technologies.

Following our successful first edition, this course will mark a revised second version, reflecting the high demand and positive feedback received previously. Building on the foundation laid by the inaugural course, we aim to delve deeper into the realm of single-cell technologies. This time, we are dedicated to exploring a broader spectrum of cutting-edge methodologies. These advancements promise to further enhance the capacity of researchers in Latin America, equipping them with the knowledge and skills needed to leverage the full potential of single-cell technologies in their scientific endeavours.

## Target Audience

Early and mid-career scientists, PhD students, and postdoctoral researchers based in Latin American countries who are engaged or planning to engage in single-cell research.

## Prerequisites

We recommend that participants be familiar with the basics of genomics and molecular Biology. The hands-on bioinformatics modules will be taught using R. Therefore, participants are required to have some familiarity with the R language. This will be essential for participants to fully benefit from the course. All candidates will be requested to undertake pre-course activities to gain the fundamental knowledge needed during the course.

The course will be taught in English but there will be support from Portuguese and Spanish-speaking instructors.

## Course Content

This hands-on laboratory and bioinformatics course will offer a series of lectures and practical sessions including the following topics:

- Introduction to single-cell sequencing: applications and analysis
- Single-cell RNA sequencing methods: detailed state-of-the-art molecular biology protocols for sample preparation
- Hands-on single-cell techniques: from cell preparation to data analysis
- Best practices for planning and executing single-cell experiments, quality control and troubleshooting
- Bioinformatics training: workflows and tools for data processing, visualization, and cellular structure identification 
- Group projects: bioinformatics analyses and grant writing

## Learning Outcomes

At the end of this course, participants will be able to:

- Compare single-cell RNA sequencing (scRNA-seq) methodologies to identify suitable applications. 
- Plan and perform scRNA-seq laboratory techniques and data analysis to obtain high-quality outputs. 
- Apply R scripting and publicly available data repositories to analyse scRNA-seq data.
- Design and execute a bioinformatics analysis pipeline for scRNA-seq data. 
- Interpret the outcomes of a single-cell RNA-seq pipeline in light of a biological question.

## Scientific Organising Committee
- [Patricia Abrão Possik](https://www.researchgate.net/profile/Patricia-Possik-2), National Institute of Cancer (INCA), Brazil
- [David Adams](https://www.sanger.ac.uk/person/adams-david/), Wellcome Sanger Institute, UK
- [Mariana Boroni](https://www.gov.br/inca/pt-br/assuntos/pesquisa/pesquisa-basica-e-experimental/bioinformatica-e-biologia-computacional), National Institute of Cancer (INCA), Brazil

## Instructors & Assistants
- [Vinicíus Maracajá-Coutinho](https://integrativebioinformatics.me/), University of Chile, Chile
- [Patricia Severino](https://www.researchgate.net/profile/Patricia-Severino), Albert Einstein Research and Education Institute, Brazil
- [Lia Chappell](https://www.sanger.ac.uk/person/chappell-lia/), Wellcome Sanger Institute, UK
- [Benilton de Sá Carvalho](http://lattes.cnpq.br/0897291971174045), Unicamp, Brazil
- [Yesid Cuesta Astroz](https://scholar.google.es/citations?user=q0hiAjEAAAAJ&hl=es), University of Antioquía, Colombia
- [Anita Scoones](https://www.earlham.ac.uk/profile/anita-scoones), Earlham Institute, United Kingdom
- [Danielle Carvalho](https://br.linkedin.com/in/danielle-carvalho-2aa894197?trk=public_profile_browsemap_profile-result-card_result-card_full-click), National Institute of Cancer (INCA), Brazil
- [Flavia Aguiar](https://br.linkedin.com/in/flavia-aguiar-7412b8211), National Institute of Cancer (INCA), Brazil
- Domenica Marchese, University of Chile, Chile
- Laura Leaden, Albert Einstein Research and Education Institute, Brazil
- [Leandro Santos](https://sites.google.com/view/bioinformaticainca-en/about-us), National Institute of Cancer (INCA), Brazil
- [Gabriela Rapozo](https://www.researchgate.net/profile/Gabriela-Rapozo), National Institute of Cancer (INCA), Brazil
- [Adolfo Rojas](https://www.linkedin.com/in/adolfo-rojas-hidalgo-4b2a85164/), University of Chile, Chile
- [Jacqueline Boccacino](https://www.sanger.ac.uk/person/boccacino-jacqueline/), Wellcome Sanger Institute, UK
- Annie Squiavinato, National Institute of Cancer (INCA), Brazil
- Nayara Tessarollo, National Institute of Cancer (INCA), Brazil

## Wellcome Connecting Science Team
- [Alice Matimba](https://uk.linkedin.com/in/alice-matimba-8805177), Head of Training and Global Capacity
- [Liã Bárbara Arruda](https://www.wellcomeconnectingscience.org/person/barbara-arruda-lia/), Education Developer
- [Isabela Malta](https://www.linkedin.com/in/isabela-malta-754503162/), Assistant Global Training Manager
- [Karon Chappell](https://coursesandconferences.wellcomeconnectingscience.org/about-us/the-team/), Event Organiser
- [Cassandra Soo](https://uk.linkedin.com/in/cassandra-claire-soo-b3783277/ms?trk=people-guest_people_search-card), Laboratory Courses Manager
- [Aaron Dean](https://uk.linkedin.com/in/aaron-dean-5b5a21163), Laboratory Technical Officer
- [Martin Asltett](https://coursesandconferences.wellcomeconnectingscience.org/about-us/the-team/), Informatics Manager
- [Vaishnavi Vikas Gangadhar](https://www.wellcomeconnectingscience.org/person/gangadhar-vaishnavi/), Informatics Technical Officer

******

## Bioinformatics Material

### Module 1: Single Cell Resources
This module provides an overview of key resources and tools available for single-cell research, including databases, repositories, and software platforms that facilitate the exploration and analysis of single-cell data.  
- [Slides](https://docs.google.com/presentation/d/189-cTq1yZYZtwIOWm740hge-5simQcqg3TmEot6npY0/edit?usp=drive_link)

### Module 2: Hands-on Databases
n this hands-on module, participants will explore various databases relevant to single-cell research, learning how to access, navigate, and extract valuable information for their studies.  
- [Google Colab Notebook](https://colab.research.google.com/github/WCSCourses/Single_Cell_Genomics_2024/blob/main/modules/Module_01_02_DataBases.ipynb)

### Module 3: Data Processing
This module covers the fundamental steps of single-cell data processing, including quality assessment, read alignment, and the generation of expression matrices. Participants will gain practical experience with common data processing pipelines.  
- [Google Colab Notebook](https://colab.research.google.com/github/WCSCourses/Single_Cell_Genomics_2024/blob/main/modules/Module_03_Processing_RawSeqs_CellRanger.ipynb)
  
- [Slides](https://docs.google.com/presentation/d/1i_jUWjEkZxL_Oaa4ys1y0h9RXeRq1BUi/edit?usp=drive_link&ouid=110642948640028618619&rtpof=true&sd=true)

### Module 4: Single-cell RNASeq Data Structure
Participants will learn about the structure and organization of single-cell RNA sequencing data. The module covers data formats, annotation standards, and how to manage and manipulate large-scale datasets.  
- [Google Colab Notebook](https://colab.research.google.com/github/WCSCourses/Single_Cell_Genomics_2024/blob/main/modules/Module_04_scRNAseq_Data_Structure.ipynb)

### Module 5: Data Quality Control
This module focuses on techniques for assessing and ensuring the quality of single-cell RNA sequencing data. Topics include identifying and mitigating sources of technical noise, filtering out low-quality cells, and standardizing quality control metrics.  
- [Google Colab Notebook](https://drive.google.com/file/d/1Sp4feBQOZ0aM3qkFHQh1K9WzJpiTut22/view?usp=drive_link)
- [Slides](https://docs.google.com/presentation/d/1HW_Wi2RMOVU_-CrVuliHPR5TS8d3fGX5/edit?usp=drive_link&ouid=109970662236337661286&rtpof=true&sd=true)
  
### Module 6: Data Normalization and Clustering
n this module, participants will learn methods for normalizing single-cell RNA sequencing data to account for technical variability. The module also covers clustering techniques to identify distinct cell populations within a dataset.  
- [Google Colab Notebook](https://drive.google.com/file/d/1Sp4feBQOZ0aM3qkFHQh1K9WzJpiTut22/view?usp=drive_link)
- [Slides](https://benilton.github.io/wcs-rio/norm_clust.html)

### Module 7: Differential Expression and Cell Type Annotation
This module teaches methods for identifying differentially expressed genes between cell populations and annotating cell types based on gene expression profiles. Techniques for statistical testing and functional annotation will be discussed.
- [Google Colab Notebook](https://drive.google.com/file/d/1Sp4feBQOZ0aM3qkFHQh1K9WzJpiTut22/view?usp=drive_link)
- [Slides](https://benilton.github.io/wcs-rio/de_annot.html)

### Module 8: Functional Analysis
Participants will explore various approaches to functional analysis of single-cell RNA sequencing data, including pathway analysis, gene set enrichment, and the integration of external biological knowledge to interpret cellular functions.
- [Google Colab Notebook](https://drive.google.com/file/d/1qztBtdAJSTjQW5FGokcuaT-F59VZnF0a/view?usp=share_link)
- Slides

### Module 9: scRNA-seq Dataset Integration
This module covers strategies for integrating multiple single-cell RNA sequencing datasets to achieve a comprehensive understanding of cellular heterogeneity. Participants will learn about batch effect correction, data harmonization, and integration algorithms.
- [Google Colab Notebook](https://drive.google.com/file/d/1g1zCMmMcgxHwLsCU7mttW4tcss6CmsXv/view?usp=drive_link)
- Slides

### Module 10: Integration Benchmarks
In this module, participants will evaluate different integration methods through benchmark datasets. They will learn how to assess the performance of integration techniques and select the appropriate method for their specific research questions.
- [Google Colab Notebook](https://drive.google.com/file/d/13iUa1SdEyPDYQYUaDC0sSJ0Ms5FIKfUJ/view?usp=drive_link)
- [Slides](https://docs.google.com/presentation/d/1f2jQuYD9TeeXhYZw66ZakoSxyfmkDlr0xpq_prT0LsI/edit?usp=sharing)
- [Data to be used](https://drive.google.com/file/d/14D0a5MctYveP8N77__jzJw8FEt5IdMIs/view?usp=drive_link)
- [Data with all integrations](https://drive.google.com/file/d/1Z8aXT7vGtfIyIqVi9YLSd2XI6C3SD2iH/view?usp=drive_link)

### Project: Project Development
We believe in hands-on learning and want to put your newly acquired bioinformatics skills to practice! At the end of our exciting week of classes, we've prepared a short project for you to dive into. This project will allow you to apply the knowledge gained throughout the course and further reinforce your understanding.

Please access the project instructions [here](https://drive.google.com/file/d/1dGEdcvXCZfP4hY0dBerNiFDNjrLgMXeq/view?usp=drive_link).

Feel free to explore, experiment, and ask questions as you work on the project. Remember, this is a fantastic opportunity to challenge yourself and gain practical experience in the fascinating world of single-cell transcriptomic RNA-seq data analysis.

### Authorship and Acknowledgments
This comprehensive material has resulted from collaborative efforts since 2021. It has been successfully employed in numerous courses organized by esteemed institutions like the Human Cell Atlas, the LatinCells initiative, and Wellcome Connecting Sciences. We extend our heartfelt gratitude to all the individuals listed below, who have actively contributed to the development and refinement of this material over the years. Their dedication and expertise have been instrumental in making this resource valuable for the bioinformatics community.
We appreciate the continuous support and feedback from participants, mentors, and institutions that have made this endeavor possible. Together, we strive to advance the understanding and application of single-cell genomics in Latin America and the Caribbean.

List of Contributors - Listed Alphabetically:

- Adolfo Rojas
- Benilton S. Carvalho
- Carlos Alberto Oliveira de Biagi Júnior
- Cesar Prada
- Cristóvão Antunes
- Erick Armingol
- Gabriela Guimarães
- Jacqueline Marcia Boccacino
- Leandro Santos
- Mariana Boroni
- Patricia Severino
- Raúl Arias-Carrasco
- Ricardo Khouri
- Vinicius Maracaja-Coutinho
- Yesid Cuesta

******

## Citing and Re-using Course Material

The course data are free to reuse and adapt with appropriate attribution. All course data in these repositories are licensed under the <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</a>. <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><br /> 

## Interested in attending a course?

Take a look what courses are coming up at [Wellcome Connecting Science Courses & Conference Website](https://coursesandconferences.wellcomeconnectingscience.org/our-events/).

---

For more information or queries, feel free to contact us via the [Wellcome Connecting Science website](https://coursesandconferences.wellcomeconnectingscience.org).<br /> 


Find us on socials [Wellcome Connecting Science Linktr](https://linktr.ee/eventswcs)

---

[Wellcome Connecting Science GitHub Home Page](https://github.com/WCSCourses) <br /> 
